third-party/realdds/src/dds-device-server.cpp

```diff
@@ -4,264 +4,91 @@
 #include <realdds/dds-device-server.h>
 
 #include <realdds/dds-participant.h>
+#include <realdds/dds-publisher.h>
 #include <realdds/dds-stream-server.h>
+#include <realdds/dds-notification-server.h>
 #include <realdds/dds-utilities.h>
 #include <realdds/topics/dds-topics.h>
+#include <realdds/dds-topic.h>
+#include <realdds/dds-topic-writer.h>
 
-#include <fastdds/dds/domain/DomainParticipant.hpp>
-#include <fastdds/dds/publisher/Publisher.hpp>
-#include <fastdds/dds/publisher/DataWriter.hpp>
-#include <fastdds/dds/publisher/DataWriterListener.hpp>
-#include <fastdds/dds/publisher/qos/PublisherQos.hpp>
-#include <librealsense2/utilities/concurrency/concurrency.h>
+#include <fastdds/dds/topic/Topic.hpp>
```

Comment:
Needed? probably covered by `dds-topic` or `dds-topic-writer`

Reply:
Needed because we use the Topic's `get_name()`